Tutorials/Creating-Basic-Site/Adding-Language-Variants/index.md

```diff
@@ -0,0 +1,82 @@
+---
+versionFrom: 8.0.0
+---
+
+# Adding Language Variants
+
+Now that we have a basic site set up, let's make the site multilingual by making variations of our content in one more language. For this we will make a Danish version of the `Contact Us` page.
+
+## Adding a new language
+
+To add a new language, follow these steps:
+
+1. Go to the **Settings** tab.
+2. In the **Settings** pane, go to **Languages**. The **Languages** window opens in the editor.
+3. Click **Add Language**. The **Add Language** window opens in the editor.
+4. Select a **Language** from the dropdown list. In this tutorial, we will pick Danish.
+    ![Adding the Danish language](images/adding-danish-language.png)
+5. In **Settings**, to set the new language as the:
+    1.  Defualt language for your site, toggle **Default Language**.
+    2.  Mandatory language for your site, toggle  **Mandatory Language**.
+6. Select a **Fallback Language** from the drop-down list.
+7. Click **Save**.
+
+## Enabling Language Variants on Document Types and Properties
+
+To enable language variants on document types, follow these steps:
+
+1. Go to the **Settings** tab.
+2. From the **Document Types** folder, select **Home Page**.
+3. Go to the **Permissions** tab and toggle **Allow vary by culture**
+4. Click **Save**.
+5. Go to the **Design** tab.
+6. Click on the gear icon ⚙ of the **Page Title** and toggle **Allow vary by culture**.
+7. Click **Submit**.
+    ![Allow property editor Language Variants](images/allow-varying-property-editor.png)
+8. For this tutotial, we will not make any changes to the **Body Text**. Click **Save**.
+
+To enable language variants on properties, follow these steps:
```

Comment:
Enabling language variants on properties is what was done in steps 5-8 in the previous list.

The steps that follow this sentence, ass culture and hostnames to the root note of the website. 
This sentence needs to reflect that.